service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinNotificationEmailService.kt

```diff
@@ -0,0 +1,169 @@
+            ) AS se ON b.id = se.bulletin_id AND bi.receiver_id = se.receiver_id
+            WHERE b.sent_at IS NOT NULL
+            AND bi.read_at IS NULL
+            AND g.email IS NOT NULL
+            AND email_sent_count IS NULL
+                """.trimIndent()
+            )
+                .mapTo<BulletinNotification>()
+                .toList()
+        }
+    }
+
+    fun scheduleSendingBulletinNotifications(db: Database.Connection) {
+        val bulletinNotifications = getNewBulletinNotifications(db)
+
+        var createdJobCount = 0
+        db.transaction { tx ->
+            bulletinNotifications.forEach { notification ->
+                if (!notification.receiverEmail.isNullOrBlank()) {
+                    asyncJobRunner.plan(
+                        tx,
+                        payloads = listOf(
+                            SendUnreadBulletinNotificationEmail(
+                                receiverId = notification.receiverId,
+                                bulletinId = notification.bulletinId,
+                                receiverEmail = notification.receiverEmail,
+                                language = notification.language
+                            )
+                        ),
+                        runAt = Instant.now(),
+                        retryCount = 10
+                    )
+                    createdJobCount++
+                } else {
+                    logger.warn("Could not send bulletin notification email to guardian ${notification.receiverId}: invalid email")
+                }
+            }
+        }
+    }
+
+    fun sendBulletinNotification(db: Database, msg: SendUnreadBulletinNotificationEmail) {
+        db.transaction { tx ->
+            val lang = getLanguage(msg.language)
+
+            emailClient.sendEmail(
+                traceId = "${msg.receiverId} - ${msg.bulletinId}",
+                toAddress = msg.receiverEmail,
+                fromAddress = fromAddress,
+                getSubject(lang),
+                getHtml(lang),
+                getText(lang)
+            )
+
+            // Mark as sent
+            tx.createUpdate(
+                """
+                INSERT INTO sent_email (sent_at, receiver_id, bulletin_id, type)
+                VALUES (now(), :receiverId, :bulletinId, :type::sent_email_type);
+                """.trimIndent()
+            )
+                .bind("receiverId", msg.receiverId)
+                .bind("bulletinId", msg.bulletinId)
+                .bind("type", "UNREAD_BULLETIN_REMINDER")
+                .execute()
+        }
+    }
+
+    private fun getLanguage(languageStr: String?): Language {
+        return when (languageStr) {
+            "sv", "SV" -> Language.sv
+            "en", "EN" -> Language.en
+            else -> Language.fi
+        }
+    }
+
+    private fun getSubject(language: Language): String {
+        val postfix = if (System.getenv("VOLTTI_ENV") == "staging") " [staging]" else ""
```

Comment:
Maybe something like this would be better (no reason to constrain this functionality to just staging):
```suggestion
        val postfix = if (System.getenv("VOLTTI_ENV") == "prod") "" else " [${System.getenv("VOLTTI_ENV")}]"
```